plugins/afl-scoreboard/manager.py

```
@@ -438,11 +380,51 @@ def _ensure_manager_updated(self, manager) -> None:
+        self.live_priority = bool(self.config.get("live_priority", False))
+
+        # Snapshot and clear in-flight update threads under the lock (a fast,
+        # non-blocking dict operation), then join them WITHOUT holding the
+        # lock -- update()'s own thread-starting path (below) also needs
+        # _config_lock, and sequential thread.join(timeout=10.0) calls while
+        # holding it would block display()/update() for up to 10s per thread.
+        with self._config_lock:
+            threads_to_drain = list(self._active_update_threads.items())
+            self._active_update_threads.clear()
+
+        for name, thread in threads_to_drain:
```

Ruff (0.15.21)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)